draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2038,19 +2038,19 @@ Internet-Draft                 ACTN POI                       July 2022
    <~~~> Link inferred by the MDSC
    {   } LTP Plug-id reported by the PNC
 
-    Figure 7 - Single-layer intra-domain Ethernet and IP link discovery
+    Figure 8 - Single-layer intra-domain Ethernet and IP link discovery
 
    In this case, the MDSC, using the plug-id information reported in the
    physical Ethernet LTPs, does not discover any cross-layer link being
    terminated by the corresponding Ethernet interface. The  MDSC may
```

Comment:
I would add: "since to discover cross-layer link MDSC should match plug-in info from physical Ethernet LTP in the router and the physical Ethernet LTP in Optical NE, and not by just the Ethernet LTPs in the router. "

Reply:
Fixed in https://github.com/FabioPeruzzini/actn-poi/pull/101/commits/8b5408c65f2c5f10fe78b4791c3a3dfd361e0fe6